hudi-client/src/test/java/org/apache/hudi/TestWriteStatus.java

```diff
@@ -49,7 +58,18 @@ public void testSuccessRecordTracking() {
     }
     assertEquals(1000, status.getFailedRecords().size());
     assertTrue(status.hasErrors());
+    assertEquals(status.getErrors().size(), 1);
     assertTrue(status.getWrittenRecords().isEmpty());
     assertEquals(2000, status.getTotalRecords());
+
+    status.setTotalRecords(1000);
+    assertEquals(status.getTotalRecords(), 1000);
+  }
+
+  @Test
+  public void testAdditional() {
```

Comment:
This test method does not have any assert, why we need to have it?

Reply:
+1

Reply:
removed